### DISCUSSION TITLE
How to change the port

### DISCUSSION THREAD

**Discussion by Mashfiqur25** [openbullet/OpenBullet2#1043] [General]
I see the default port is 5000. I wanna use a custom value for the port. How to do it ?

**Comment by openbullet** [comment]
Launch OB2 from command line and add `--urls "http://localhost:1234"` where `1234` is your desired port. If you need to access it from elsewhere, use `*` instead of `localhost`.

For example,
```bash
$ dotnet ./OpenBullet2.Web.dll --urls "http://localhost:42069"
...
[00:19:36 INF] Now listening on: http://localhost:42069
```
